### DISCUSSION TITLE
Note about prerequisites

### DISCUSSION THREAD

**Discussion by yllekz** [wg-easy/wg-easy#1147] [Ideas]
Hello, I wanted to bring to your attention a requirement for this container that often goes overlooked and appears to be undocumented.

For every Linux system I've stood up this container on, I've had to install the ``wireguard-dkms`` package before wg-easy would function. For example, on Debian systems:

```
sudo apt-get install wireguard-dkms
#Requires a reboot at this point
```

Meanwhile, certain Synology systems require a different prereq called [synology-wireguard](https://github.com/runfalk/synology-wireguard) and it varies based on model.

If one tries to stand up this container without the prereqs, they may find that they will not be able to connect.

It might be a good idea to note this somewhere in the docs. I did not see it in the readme nor the wiki.

**Comment by kaaax0815** [comment]
This is only needed for kernels which don't natively support WireGuard. It is not needed for any Kernel >= 5.6
